scene/gui/code_edit.cpp

```diff
@@ -2818,6 +2908,15 @@ TypedArray<String> CodeEdit::_get_delimiters(DelimiterType p_type) const {
 	return r_delimiters;
 }
 
+/* Code Region */
+String CodeEdit::_get_code_region_tag(String p_tag) const {
+	TypedArray<String> comment_delimiters = get_comment_delimiters();
```

Comment:
Might be an idea to check if it's line `line_only`, or search for one that is, else it will need a closing symbol.